drivers/net/usb/smsc95xx.c

```diff
@@ -2082,7 +2082,9 @@ static struct sk_buff *smsc95xx_tx_fixup(struct usbnet *dev,
 					 struct sk_buff *skb, gfp_t flags)
 {
 	bool csum = skb->ip_summed == CHECKSUM_PARTIAL;
-	int overhead = csum ? SMSC95XX_TX_OVERHEAD_CSUM : SMSC95XX_TX_OVERHEAD;
+	unsigned int align_bytes = (4 - ((uintptr_t)skb->data & 0x3)) & 0x3;
```

Comment:
The first `& 0x3` is unnecessary.

Reply:
Pretty sure the 4 is unnecessary too. ``(-x)&3`` should be enough...

Reply:
Indeed.

Reply:
GCC agrees, disassembly is the same

Reply:
That comment needs an emoji to disambiguate the snark level.

Reply:
Or a push